.chloggen/gcp-v0350-update.yaml

```diff
@@ -0,0 +1,22 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: googlecloudexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: New release of GCP exporter
+
+# One or more tracking issues related to the change
+issues: []
```

Comment:
Please update this field

Reply:
Good to know, that wasn't clear. Opened a tracking issue instead